tests/fixture/tmpnet/defaults.go

```diff
@@ -19,6 +19,15 @@ const (
 	DefaultNetworkStartTimeout = 2 * time.Minute
 	DefaultNodeInitTimeout     = 10 * time.Second
 	DefaultNodeStopTimeout     = 5 * time.Second
+
+	// Minimum required to ensure connectivity-based health checks will pass
+	DefaultNodeCount = 2
```

Comment:
probably out of scope here but should this be MinNodeCount? And should we assert that we never have a single node network? Not sure if the latter is too restrictive for our testing.